next.clayui.com/content/docs/migration/migrate-the-clay-components-from-v2-to-v3.md

```diff
@@ -103,7 +102,7 @@ const [alerts, setAlerts] = useState([]);
 
 ## ClayButton
 
--   Removed icon dependency within button itself
+-   Removed icon dependency within `ClayButton` itself. However, was created a high-level component called `ButtonWithIcon` that receives a symbol and creates a button monospaced with a icon.
```

Comment:
Since we have ClayButtonWithIcon and thus including the dependency again, we can just remove this line entirely.